R/fastMapGenerate.R

```diff
@@ -23,7 +23,7 @@
 #' @examples
 #' \dontrun{
 #' fastMapGenerate("hgnc_complete_set.txt", "symbol", "uniprot_ids",
-#'                 type = "UniProt", outName = "fastMapUniProt.rds")
+#'                 type = "UniProt", outputName = "fastMapUniProt.rds")
 #' }
```

Comment:
IDK - haven't looked at that yet. Doesn't he need to be a collaborator on master to be able to handle issues? Perhaps just write a note on Riot ...